pkg/segment/aggregations/evalaggs.go

```diff
@@ -600,6 +600,35 @@ func PerformAggEvalForCardinality(measureAgg *structs.MeasureAggregator, strSet
 	return float64(len(strSet)), nil
 }
 
+func PerformAggEvalForList(measureAgg *structs.MeasureAggregator, strList []string, fieldToValue map[string]utils.CValueEnclosure) ([]string, error) {
```

Comment:
nit: it would be good to rename `strList` to something more meaningful, as `strList` just repeats its type

Reply:
Changed to currentList